ros_ws/src/simulation/racer_control/include/drive_param_converter.h

```diff
@@ -0,0 +1,102 @@
+#pragma once
+#include <string>
+#include <cmath>
+#include <ros/ros.h>
+
+#include <drive_msgs/drive_param.h>
+#include <std_msgs/Float64.h>
+
+
+/**
+ * @brief 
+ * 
+ */
+struct Angles
+{
+    double angle1;
+    double angle2;
+};
+
+/**
+ * @brief 
+ * 
+ */
+class DriveParamConverter
+{
+    public:
+    /**
+     * @brief Construct a new Drive Pram Converter object
+     * 
+     */
+    DriveParamConverter();
+    /**
+     * @brief 
+     * 
+     * @param parameters 
+     */
+    void convertDriveParametersCallback(const drive_msgs::drive_param::ConstPtr& parameters);
+    
+
+    private:
+    /**
+     * @brief 
+     * 
+     * @param angle Angle of the center of the front axis.
+     * @return Angles One Ackermann angle for each front wheel
+     */
+    Angles calculateSteeringAngles(const double& angle);
```

Comment:
The const ref angle seems unnecessary complex. In worst case it adds a unnecessary indirection to the function call.